src/ddprof_cli.cc

```diff
@@ -297,10 +297,14 @@ int DDProfCLI::parse(int argc, const char *argv[]) {
       app.add_flag("--help_extended", help_extended, "Show extended options")
           ->group(""));
   extended_options.push_back(
-      app.add_option("--socket", socket,
-                     "Profiler's IPC socket, as a file descriptor")
+      app.add_option("--socket", socket_path, "Profiler socket path")
```

Comment:
Minor: we could give a more detailed explanation of why we establish this socket.

Reply:
As a user I would be confused with what I can do with this.
The overview is great.

Reply:
I guess these are more developer options. We do not use them apart from testing use cases. Would something like:
`Override the automatically created socket with a specific path`
Be better ?